controller/src/main/java/io/pravega/controller/server/rest/RESTServer.java

```diff
@@ -67,7 +69,7 @@ protected void startUp() {
         long traceId = LoggerHelpers.traceEnterWithContext(log, this.objectId, "startUp");
         try {
             log.info("Starting REST server listening on port: {}", this.restServerConfig.getPort());
-            channel = NettyHttpContainerProvider.createServer(baseUri, resourceConfig, false);
+            httpServer = GrizzlyHttpServerFactory.createHttpServer(baseUri, resourceConfig, true);
```

Comment:
Here 3rd param is to start the server. Setting to true here otherwise it will only configure the server and we need to explicitly call httpServer.start() to start listening.